website/blog/2022-03-08-aluminium.md

```diff
@@ -0,0 +1,441 @@
+Sublime Text and Eclipse. Metals is developed at the
+[Scala Center](https://scala.epfl.ch/) and [VirtusLab](https://virtuslab.com)
+with the help from [Lunatech](https://lunatech.com) along with contributors from
+the community.
+
+## TL;DR
+
+Check out [https://scalameta.org/metals/](https://scalameta.org/metals/), and
+give Metals a try!
+
+- Reduce indexing time in monorepos on MacOS
+- Display build target info
+- View source jar files as virtual docs
+- Improve completions for Scala 3
+- [Test Explorer] Detect and run single tests for JUnit4
+
+## Reduce indexing time in monorepos on MacOS
+
+Thanks to changes made by [Duhemm](https://github.com/Duhemm), Metals on MacOS can now index monorepo workspaces much faster than before. After changes, we observed at times **over 10 times faster** indexing times in the workspaces where only a few submodules needed to be imported.
+
+Here are a few examples of indexing improvement. They are not actual benchmarks of any kind, but more of rough estimations of improvements in a couple of example repos.
+
+| Before      | After      |
+| ----------- | ---------- |
+| \>10minutes | ~30seconds |
+| ~40seconds  | ~4s        |
+
+However, take into mind that indexing speedup heavily depends on your repository structure. The more imported modules which don't need to be imported, the greater the speed improvement is.
+
+If you are interested in details you can check out the related [pull request](https://github.com/scalameta/metals/pull/3665).
+
+## Display build target info
+
+Thanks to the [Arthurm1](https://github.com/Arthurm1) Metals is now able to display all important information about modules.
+This brand new feature gathers in one view information such as:
+
+- javac and scalac options
+- dependent modules
+- projects classpath
+- and many more
+
+![display-build-target-info](https://i.imgur.com/XGyJEsl.gif)
+
+## [vscode] View source jar files as virtual docs
+
+Previously, when the client wanted to browse files in source jars source, these files were extracted and saved in the `metals/readonly/dependencies` directory. With the help of [virtual documents](https://code.visualstudio.com/api/extension-guides/virtual-documents) Metals can show you dependencies in readonly files without unnecessary copying.
+Thanks [Arthurm1](https://github.com/Arthurm1) for this feature!
+
+Together with Metals tab, this feature could be used to browse through your dependencies' sources. Just run the `Metals: Reveal Active File in Side Bar` command and browse through both dependencies and source code seamlessly.
+
+![virtual-docs-navigation](https://imgur.com/HsuW8Hn.gif)
+
+## Improve completions for Scala 3
+
+Completion suggestions for different Scala keywords now work with most of the Scala 3 keywords. This includes for example `given` and `enum`, it should also work even if defining things in toplevel without a wrapping class or object.
+![keyword-completions](https://i.imgur.com/4BUxCDK.gif)
+
```

Comment:
```suggestion
Another improvement for Scala 3 completions is better support for showing scope completions, when writing in an empty line. Previously, we would not show correct completions when no identifier was specified.

"""scala
object Foo:
  def bar: Int = 42
  def baz: Int =
    val x = 1
    val y = 2
  @@

In the above situation with cursor position indicated by `@@` we will now properly show `bar` and `baz` completions.

```


instead of `"""` we should use triple backtick